cli/bin/lsp.js

```diff
@@ -1,23 +1,14 @@
-const path = require("path");
-const { execSync } = require("child_process");
-
-const grainc = path.join(__dirname, "grainc.exe");
+const exec = require("./exec");
 
 // call the compiler, passing stdio through so the compiler gets the source code on stdin
 // and we get the compiler output in stdout
 // we still take the file name so we have it available
 
 module.exports = (file, options) => {
   try {
-    execSync(
-      `${grainc} --stdlib=${options.stdlib} ${
-        options.cflags ? options.cflags : ""
-      } -g --lsp ${file}`,
-      { stdio: "inherit" }
-    );
+    exec(`-g --lsp ${file}`, options, { stdio: "inherit" });
     process.exit();
   } catch (e) {
-    console.log(e);
```

Comment:
@ospencer this uses `stdio: inherit` already, so we don't need to log the err here.